kong/runloop/wasm.lua

```diff
@@ -72,6 +73,20 @@ local FILTER_META_SCHEMA = {
   type = "object",
   properties = {
     config_schema = json_schema.metaschema,
+    metrics = {
+      type = "object",
+      properties = {
+        label_patterns = {
+          type = "array",
+          items = {
+            properties = {
+              label = { type = "string" },
+              pattern = { type = "string" },
+            }
+          }
```

Comment:
(please double check me for correctness)

1. the code that consumes this JSON assumes that each of `label_patterns[]` has both `label` and `pattern` properties, so they should be required.
2. nit: add explicit `type = "object"`

```suggestion
          items = {
            type = "object",
            required = { "label", "pattern" },
            properties = {
              label = { type = "string" },
              pattern = { type = "string" },
            }
          }
```

We have an integration test case which asserts that filter metadata is validated according to the JSON schema at init ([link](https://github.com/Kong/kong/blob/d4ab528fa2414d996861a43e58406c02b4978157/spec/02-integration/20-wasm/09-filter-meta_spec.lua#L403-L416)). Should be easy to copy/paste and create another test case for metrics, if you'd like.